src/sentry/static/sentry/app/views/stream/organizationSavedSearchSelector.jsx

```diff
@@ -64,12 +69,109 @@ export default class OrganizationSavedSearchSelector extends React.Component {
           }
         >
           {this.renderList()}
+          <StyledMenuItem divider={true} />
+          <ButtonBar>
+            <SaveSearchButton query={query} orgId={orgId} />
+          </ButtonBar>
```

Comment:
In the spec only manager and owner roles are allowed to create saved searches.

Reply:
I imagine this should include admin as well? I'll add a check for `project:write`